frontend/src/app/footer/footer.component.html

```diff
@@ -34,7 +34,7 @@ <h3 class="nav-title">Strefa wolontariusza</h3>
             <a class="nav-link" href="{{ djangoRoot }}/offers">Lista aktualnych ofert</a>
             <a class="nav-link" href="{{ djangoRoot }}/offers/archived">Archiwum ofert</a>
             <a class="nav-link" href="{{ djangoRoot }}/organizations">Lista organizacji i instytucji</a>
-            <a class="nav-link" href="{{ djangoRoot }}/faq-volunteers">FAQ dla wolontariuszy</a>
+            <a class="nav-link" [routerLink]="['/faq-volunteers']">FAQ dla wolontariuszy</a>
```

Comment:
I know that there is an issue with `routerLink`:
```
HeadlessChrome 0.0.0 (Linux 0.0.0) AppComponent should create the app FAILED
	Can't bind to 'routerLink' since it isn't a known property of 'a'. (""{{ djangoRoot }}/organizations">Lista organizacji i instytucji</a>
	            <a class="nav-link" [ERROR ->][routerLink]="['/faq-volunteers']">FAQ dla wolontariuszy</a>
	            <a class="nav-link" href="{{"): ng:///DynamicTestModule/FooterComponent.html@36:32
	Error: Template parse errors:
	    at syntaxError (webpack:///~/@angular/compiler/@angular/compiler.es5.js:1694:21 <- src/test.ts:38828:34)
	    at TemplateParser.Array.concat.TemplateParser.parse (webpack:///~/@angular/compiler/@angular/compiler.es5.js:12937:0 <- src/test.ts:50071:19)
	    at JitCompiler.Array.concat.JitCompiler._compileTemplate (webpack:///~/@angular/compiler/@angular/compiler.es5.js:27133:0 <- src/test.ts:64267:39)
	    at webpack:///~/@angular/compiler/@angular/compiler.es5.js:27052:47 <- src/test.ts:64186:62
	    at Set.forEach (native)
	    at JitCompiler.Array.concat.JitCompiler._compileComponents (webpack:///~/@angular/compiler/@angular/compiler.es5.js:27052:0 <- src/test.ts:64186:19)
	    at webpack:///~/@angular/compiler/@angular/compiler.es5.js:26953:0 <- src/test.ts:64087:19
	    at Object.then (webpack:///~/@angular/compiler/@angular/compiler.es5.js:1683:32 <- src/test.ts:38817:148)
	    at JitCompiler.Array.concat.JitCompiler._compileModuleAndAllComponents (webpack:///~/@angular/compiler/@angular/compiler.es5.js:26951:0 <- src/test.ts:64085:26)
	    at JitCompiler.Array.concat.JitCompiler.compileModuleAndAllComponentsAsync (webpack:///~/@angular/compiler/@angular/compiler.es5.js:26883:0 <- src/test.ts:64017:37)
```
but it's common for all pull requests about moving static pages, so I will try to resolve it or provide a solution.

Reply:
@magul, the solutions are already there. Routing needs to be configured in these tests i.e. in one of the ways proposed in the accepted answer here https://stackoverflow.com/questions/39577920/angular-2-unit-testing-components-with-routerlink. These tests need to be fixed just like mentioned there and I don't think you'll want to "fix" it somehow globally.